pysatModels/utils/extract.py

```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 # Copyright (C) 2019, AGB & pysat team
 # Full license can be found in License.md
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
```

Comment:
Why?  This looks so ugly with a space there.

Reply:
E265: block comment should start with '# '

Keeping flake8 happy.  I don't think there is a way to distinguish between `#Copyright` and `#---`. E265 wants to see a space after #.  If I drop the check for one, it drops for the other.

Reply:
That's a shame.